fili-core/src/main/java/com/yahoo/bard/webservice/data/HttpResponseMaker.java

```diff
@@ -95,18 +95,18 @@ public HttpResponseMaker(ObjectMappersSuite objectMappers, DimensionDictionary d
      *
      * @param resultSet  The result set being processed
      * @param responseContext  A meta data container for the state gathered by the web container
-     * @param responseFormatType  The format in which the response should be returned to the user
-     * @param uriInfo  UriInfo of the request
+     * @param apiRequest apiRequest needed for serialization writer
```

Comment:
2 spaces between param name and param desc. Capitalize first word in description